src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -119,7 +119,8 @@
         DELAYED_FOR_WIFI,
         LOCAL_FILE_NOT_FOUND,
         SERVICE_UNAVAILABLE,
-        SPECIFIC_SERVICE_UNAVAILABLE
+        SPECIFIC_SERVICE_UNAVAILABLE,
+        SPECIFIC_UNSUPPORTED_MEDIA_TYPE
```

Comment:
No, I'm wrong in that last point; this code is only set if there is an specific message. The problem is that no special result code is set when there is no specific message.

Let's leave it for the moment, we'll resume the discussion on the naming of ```SPECIFIC_```s when we aboard the generalized storage of remote messages, that has been taken out of the associated issue for now.